_includes/footer.html

```diff
@@ -16,5 +16,5 @@
     Creative Commons Attribution-ShareAlike 4.0 International License
   </a>.
 
-  &copy; 2015 The Signal K Project. | Design: <a href="http://html5up.net">HTML5 UP</a>
+  &copy; 2018 The Signal K Project. | Design: <a href="https://www.netaction.de/">Thomas Netaction Schmidt</a>
```

Comment:
One other thing I just noticed: Tom seems to spell his handle "netAction", lowercase n, uppercase A. It's also common practice to put quotes around handles in the middle of names.